sup3r/bias/qdm.py

```diff
@@ -69,9 +63,11 @@ def __init__(self,
                  match_zero_rate=False,
                  n_quantiles=101,
                  dist='empirical',
-                 relative=None,
+                 relative=True,
                  sampling='linear',
                  log_base=10,
+                 n_time_steps=24,
+                 window_size=120,
```

Comment:
I think this class needs a few more arguments ;)

Reply:
Haha too much bloat? Possibly, but for now we just need to be able to modify these from config.